input/pagecontent/changes.md

```diff
@@ -1,6 +1,16 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01-24
+- Expected Publication status: Normative
```

Comment:
AU Core R1 will not be normative  - it'll will be subject to further feedback and refinement.